freqtrade/freqtradebot.py

```diff
@@ -607,17 +610,19 @@ def handle_timedout_limit_sell(self, trade: Trade, order: Dict) -> bool:
         # TODO: figure out how to handle partially complete sell orders
         return False
 
-    def execute_sell(self, trade: Trade, limit: float) -> None:
+    def execute_sell(self, trade: Trade, limit: float, sellreason: SellType) -> None:
         """
         Executes a limit sell for the given trade and limit
         :param trade: Trade instance
         :param limit: limit rate for the sell order
+        :param sellrason: Reaseon the sell was triggered
```

Comment:
sellrason. typo.

Reply:
2, actually - Reaseon is not a word in any language i speak ...